storagemarket/types.go

```diff
@@ -411,10 +411,10 @@ type StorageClient interface {
 	// ListDeals lists on-chain deals associated with this provider
 	ListDeals(ctx context.Context, addr address.Address) ([]StorageDeal, error)
 
-	// ListInProgressDeals lists deals that are in progress or rejected
-	ListInProgressDeals(ctx context.Context) ([]ClientDeal, error)
+	// ListLocalDeals lists deals that are in progress or rejected
+	ListLocalDeals(ctx context.Context) ([]ClientDeal, error)
 
-	// ListInProgressDeals lists deals that are in progress or rejected
+	// ListLocalDeals lists deals that are in progress or rejected
```

Comment:
not your fault, but this comment looks very wrong :) (it already was very wrong :)

also, this should also become GetLocalDeal... sorry!